### PR TITLE
Edit phases

### PR DESCRIPTION
**Pull request by eaj0003**

### CODE REVIEW

File: SchedulerGUI/Models/PassOrbit.cs [hunk 1]
```diff
@@ -39,44 +42,36 @@ public PassOrbit(string name, DateTime startTime, DateTime endTime)
...
         {
             this.PassPhases = new List<IPassPhase>();
             DateTime startTime = this.StartTime;
-            double duration = (this.EndTime - this.StartTime).TotalSeconds / Enum.GetValues(typeof(PhaseType)).Length;
+            int numPhases = Enum.GetValues(typeof(PhaseType)).Length;
+            double duration = (this.EndTime - this.StartTime).TotalSeconds / numPhases;
 
             var enums = Enum.GetValues(typeof(PhaseType)).Cast<PhaseType>().Where(x => x != PhaseType.Encryption);
+            double phasePower = TOTALPOWER / numPhases;
+            double phaseEnergy = TOTALENERGY / numPhases;
```
Comment (alexdillon) [inline SchedulerGUI/Models/PassOrbit.cs:61]: Are phasePower and TOTALPOWER still used?

**Review by alexdillon — APPROVED** [review]
I would consider removing those remnants of power variables, but beyond that, I believe this fully encapsulates all the tasks discussed. Thanks!

File: SchedulerGUI/Models/PassOrbit.cs (hunk 1 above)
Comment (eaj0003) [reply SchedulerGUI/Models/PassOrbit.cs:61]: No - good catch. I removed those. I also made the max values for energy and bytes random nums, and added them to the edit view.

Comment (alexdillon) [reply SchedulerGUI/Models/PassOrbit.cs]: Awesome, works for me!

**Review by alexdillon — APPROVED** [review]
Thanks!

**Pull request merged**